external/mmsegmentation/segmentation_tasks/extension/utils/pipelines.py

```diff
@@ -40,8 +44,8 @@ def __init__(self, to_float32: bool = False):
 
     @check_input_parameters_type()
     def __call__(self, results: Dict[str, Any]):
-        dataset_item = results['dataset_item']
-        img = dataset_item.numpy
+        # Get image (possibly from cache)
+        img = get_image(results, _CACHE_DIR.name, to_float32=self.to_float32)
```

Comment:
If we rename `get_image` function to something like `get_image_cached`, the comment above won't be required.

Reply:
That can be conflicted `load_image_from_cache` in data_utils.py meaningfully. I think it will be better to remove comment.